contrib/wo_cellarea.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+
+"""
+Upward Ocean Mass Transport (wo)
+================================
+
+wo -> wmo cmorization
+
+Step.1 convert nodes to levels
+  - wo (time, nodes_3d=3668773)
+  - griddes.cell_area(ncells=126859)
+
+  Transform `wo` to (time, level, nodes_2d)
+  As nodes_2d and ncells have same dimensional values, cell_area can be applied
+
+Step.2 Apply cell_area calculations
+
+  `wo` * cell_area *  (reference density 𝜌0=1035 kg m−3)
+"""
+
+import cf_xarray.units  # noqa: F401
+import pint_xarray
+import xarray as xr
+
+import pymorize.fesom_1p4
+
+
+def nodes_to_levels(data, rule):
+    mesh_path = rule.get("mesh_path")
+    if mesh_path is None:
+        raise ValueError(
+            "Set `mesh_path` path in yaml config."
+            "Required for converting nodes to levels"
+        )
+    return pymorize.fesom_1p4.nodes_to_levels(data, rule)
+
+
+def cellarea(data, rule):
```

Comment:
I think a more descriptive name might be useful:
```suggestion
def weight_by_cellarea_and_density(data, rule):
```